fix(upgrade): replaces setAngularLib and getAngularLib with setAngula…

vsavkin opened the pull request:
…rJSGlobal and getAngularJSGlobal

The current names are confusing because Angular should refer to the latest version of the framework.

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


**What kind of change does this PR introduce?** (check one with "x")
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)



**What is the new behavior?**



**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...


**Other information**:

gkalpak commented on packages/upgrade/src/common/angular1.ts:
```diff
@@ -228,22 +228,36 @@ try {
+ * @deprecated Use {@link setAngularJSGlobal} instead.
+ */
+export function setAngularLib(ng: any): void {
+  angular = ng;
```
Nit: `angular = ng` --> `setAngularJSGlobal(ng)` (so they don't get out-of-sync)

gkalpak commented on packages/upgrade/src/common/angular1.ts:
```diff
@@ -228,22 +228,36 @@ try {
 
 /**
- * Returns the current version of the AngularJS library.
+ * Returns the current AngularJS global.
```
The `global` is a little misleading, because it is not necessarily the global one (it's just an internal reference used by `ngUpgrade`). I don't feel strongly about it though.

vsavkin replied:
You are right: it is not global, but it is a singleton. I used `global` cause I think most devs will intuitively understand what it is.

I'm fine with changing the name though.

gkalpak replied:
How about `setAngularJSThing()`? 😛
I don't have a better alternative, so unless someone comes up with a better one, I am fine keeping `Global` 😕

gkalpak commented on packages/upgrade/static/public_api.ts:
```diff
@@ -12,7 +12,7 @@
  * Entry point for all public APIs of the upgrade/static package, allowing
  * Angular 1 and Angular 2+ to run side by side in the same application.
  */
-export {getAngularLib, setAngularLib} from './src/common/angular1';
```
We should export both versions (for as long as they are deprecated).

vsavkin replied:
Fixed

vsavkin committed:
fix(upgrade): replaces get/setAngularLib with get/setAngularJSGlobal

The current names are confusing because Angular should refer to the latest version of the framework.

gkalpak commented:
public-api pull-approvers (@IgorMinar, @mhevery), could we get a LGTM, please? :grin:
